.travis.yml

```diff
@@ -5,14 +5,17 @@ sudo: false
 cache:
   directories:
   - node_modules
-install:
-- yarn add codeclimate-test-reporter
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
We have to download something? Why isn't this in package.json?

Reply:
they don't have one

https://docs.codeclimate.com/docs/configuring-test-coverage#section-quick-guide